src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -362,60 +363,131 @@ export class EducationWizard extends React.Component {
+                        <strong>or</strong> have already earned a STEM degree
+                        and are pursuing a teaching certification.{' '}
+                        <a
+                          href="https://benefits.va.gov/gibill/docs/fgib/STEM_Program_List.pdf"
+                          onClick={() =>
+                            recordEvent({
+                              event: 'edu-navigation',
+                              'edu-action': 'see-approved-stem-programs',
+                            })
+                          }
+                        >
+                          See approved STEM programs
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                ) : (
+                  <div>
+                    <strong>
+                      To be eligible for the{' '}
+                      <a
+                        href="https://benefits.va.gov/gibill/fgib/stem.asp"
+                        onClick={() =>
+                          recordEvent({
+                            event: 'edu-navigation',
+                            'edu-action': 'stem-scholarship',
+                          })
+                        }
+                      >
+                        Edith Nourse Rogers STEM Scholarship
+                      </a>
+                      , you must meet all the requirements below.
+                    </strong>
+                    <ul>
+                      <li className="ul-styling">
+                        <b>Education benefit:</b> You're using or recently used
+                        Post-9/11 GI Bill or Fry Scholarship benefits.
+                      </li>
+                      <li>
+                        <b>STEM degree:</b>
+                        <ul className="circle-bullet ul-styling li-styling2">
+                          <li className="li-styling">
+                            You're enrolled in a bachelor’s degree program for
+                            science, technology, engineering, or math (STEM),{' '}
+                            <b>or</b>
+                          </li>{' '}
+                          <li>
+                            {' '}
+                            You've already earned a STEM bachelor’s degree and
+                            are pursuing a teaching certification.{' '}
+                            <a
+                              href="https://benefits.va.gov/gibill/docs/fgib/STEM_Program_List.pdf"
+                              onClick={() =>
+                                recordEvent({
+                                  event: 'edu-navigation',
+                                  'edu-action': 'see-approved-stem-programs',
+                                })
+                              }
+                            >
+                              See eligible degree programs
+                            </a>
```

Comment:
I'm not certain, but since this is a PDF, we may want to open it in a new tab. The `rel` attribute may not be necessary here but doesn't hurt.

```suggestion
                            <a
                              href="https://benefits.va.gov/gibill/docs/fgib/STEM_Program_List.pdf"
                              target="_blank"
                              rel="noopener noreferrer"
                              onClick={() =>
                                recordEvent({
                                  event: 'edu-navigation',
                                  'edu-action': 'see-approved-stem-programs',
                                })
                              }
                            >
                              See eligible degree programs
                            </a>
```

Reply:
If this is meant to be a download link, please check out this format that we should all be using - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/152#issuecomment-553127340

Reply:
No, it's suppose to open in a separate tab.